### PR TITLE
Fix OSD upload font

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes https://github.com/betaflight/betaflight-configurator/issues/2214

Fixes an underline issue with the visibility of variables that appeared after https://github.com/betaflight/betaflight-configurator/pull/2192

I have NOT tested it, I don't have a VTX at hand, but the debug data seems much better ;)

@Asizon if you have time, please test it and confirm if it works

### CODE REVIEW

**Comment by Asizon** [comment]
@McGiverGim Tested and works as expected, thanks for fix it!

**Review by haslinghuis — APPROVED** [review]

**Comment by KAJE1977** [comment]
Thank you very much. You guys are awesome.

File: src/js/tabs/osd.js [hunk 1]
```diff
@@ -142,37 +142,38 @@ FONT.parseMCMFontFile = function(dataFontFile) {
...
         Promise.reject(msg);
     }
-    const character_bits = [];
-    const character_bytes = [];
+    const characterBits = [];
+    const characterBytes = [];
     // hexstring is for debugging
     FONT.data.hexstring = [];
-    const pushChar = function() {
+    const pushChar = function(fontCharacterBytes, fontCharacterBits) {
```
Comment (mikeller) [inline src/js/tabs/osd.js:149]: Now that you have removed the need for `pushChar()` to be inside `parseMCMFontFile()`'s closure you can just as well move it outside of the function definition to make the code a bit easier to read.

**Review by mikeller — APPROVED** [review]
Good find!

File: src/js/tabs/osd.js (hunk 1 above)
Comment (McGiverGim) [reply src/js/tabs/osd.js]: Done!

**Review by mikeller — APPROVED** [review]

**Pull request merged**